integrations/gitlab/gitlab_integration/gitlab_service.py

```diff
@@ -233,6 +238,13 @@ async def get_all_projects(self) -> typing.AsyncIterator[List[Project]]:
             project.id: project for project in all_projects
         }
 
+    @staticmethod
+    async def async_project_language_wrapper(project: Project) -> dict[str, Any]:
+        languages = await anyio.to_thread.run_sync(project.languages)
+        project_with_languages = project.asdict()
+        project_with_languages["languages"] = languages
```

Comment:
take a look at how we set custom data on other integrations
we usually prefix it with __

Reply:
I wanted to align it to how the data is being returned in github & bitbucket, @danielsinai wdyt?